src/fastcat_devices/three_node_thermal_model.h

```diff
@@ -0,0 +1,101 @@
+#ifndef FASTCAT_THREE_NODE_THERMAL_MODEL_H_
+#define FASTCAT_THREE_NODE_THERMAL_MODEL_H_
+
+// Include external then project includes
+#include "fastcat/device_base.h"
+
+// Include c then c++ libraries
+#include <cmath>
+
+// Include external then project includes
+#include "fastcat/signal_handling.h"
+#include "fastcat/yaml_parser.h"
+#include "jsd/jsd_print.h"
+
+namespace fastcat
+{
+static constexpr size_t FC_TNTM_NUM_SIGNALS =
+    2;  // the required number of signals for this device
+static constexpr size_t NODE_3_TEMP_IDX =
+    0;  // signal index for node 3 temperature
+static constexpr size_t MOTOR_CURRENT_IDX =
+    1;  // signal index for motor current
+static constexpr double REFERENCE_TEMPERATURE =
+    20.0;  // reference temperature for calculations
```

Comment:
Consider moving inside `ThreeNodeThermalModel` since `REFERENCE_TEMPERATURE` will be visible in the fastcat ns to other devices possibly

Reply:
100000% nit: this is a lot easier on my eyes.

```
// long comment
  static constexpr size_t VARIABLE = 2.0;
 ```